### PR TITLE
[*] transfer split into two

### PR DESCRIPTION
**Pull request by dusanDrabik**

### CODE REVIEW

File: src/account.ts [hunk 1]
```diff
@@ -284,12 +284,38 @@ export class AccountApi {
      * message for recipient
      *
      * @param {number} amount
-     * @param {string} fromAccount      Name or id of account
-     * @param {string} toAccount        Name or id of account
+     * @param {string} fromAccount      Id of account
```
Comment (peter-rybar) [inline src/account.ts]: what is diiference between accountName and accountId?

**Review by peter-rybar — CHANGES_REQUESTED** [review]

File: src/account.ts (hunk 1 above)
Comment (dusanDrabik) [reply src/account.ts]: account name is md5 hash of email starts with 'u', account id is in format '1.2.X'

**Comment by dusanDrabik** [comment]
https://decentplatform.atlassian.net/browse/DGO2-173

File: src/account.ts [hunk 2]
```diff
@@ -284,12 +284,38 @@ export class AccountApi {
...
+     * @param {string} toAccountName    Name of account
      * @param {string} memo             Message for recipient
      * @param {string} privateKey       Private key used to encrypt memo and sign transaction
      */
-    public transfer(amount: number,
+    public transferToName(amount: number,
```
Comment (peter-rybar) [inline src/account.ts]: transferToName was removed

File: src/account.ts [hunk 3]
```diff
@@ -284,12 +284,38 @@ export class AccountApi {
...
+    /**
+     * Same as transferToName, bud recipient is identified via email address.
+     *
+     * @param {number} amount
+     * @param {string} fromAccount      Id of account
+     * @param {string} toAccountEmail   Receiver's account email
+     * @param {string} memo             Message for recipient
+     * @param {string} privateKey       Private key used to encrypt memo and sign transaction
+     */
+    public transferToMail(amount: number,
```
Comment (peter-rybar) [inline src/account.ts]: transferToMail was removed

**Pull request merged**